apps/qubit/modules/object/config/import/ead.yml

```diff
@@ -49,6 +49,7 @@ information_object:
     title:
       XPath: "did/unittitle[not(@type)]"
       Method:  setTitle
+      Filters:  ["array('emph' => array('QubitMarkdown' => 'eadTagToMarkdown'))"]
```

Comment:
What do you think about avoiding the need for `eval` for filters through use of something like the below markup? We use `eval` for some other cases given we need to be able to evaluation some PHP constants, but we probably won't need to with filters specifications.

    Filters:
      -
        emph:
          QubitMarkdown: eadTagToMarkdown